### ISSUE TITLE
Does not copy files whose name begins with a dot - such as .eslintrc.json, .prettierignore and .prettierrc.json

### ISSUE THREAD

**Issue by Ganevru** [cpina/github-action-push-to-another-repository#25]
Thanks for the action, it's very useful!

The problem is that files such as **.eslintrc.json**, **.prettierignore** and **.prettierrc.json** are ignored and not copied to the repository.

They are also not mentioned in the terminal:

```
Cloning destination git repository
total 52
drwx------    6 root     root          4096 Mar 23 20:43 .
drwxrwxrwt    1 root     root          4096 Mar 23 20:43 ..
drwxr-xr-x    8 root     root          4096 Mar 23 20:43 .git
-rw-r--r--    1 root     root           945 Mar 23 20:43 .gitignore
-rw-r--r--    1 root     root             0 Mar 23 20:43 .gitkeep
-rw-r--r--    1 root     root          1077 Mar 23 20:43 LICENSE
-rw-r--r--    1 root     root          1588 Mar 23 20:43 README.md
drwxr-xr-x    5 root     root          4096 Mar 23 20:43 content
-rwxr-xr-x    1 root     root          2882 Mar 23 20:43 gatsby-config.js
-rw-r--r--    1 root     root           466 Mar 23 20:43 gatsby-node.js
-rw-r--r--    1 root     root            56 Mar 23 20:43 netlify.toml
-rwxr-xr-x    1 root     root          1011 Mar 23 20:43 package.json
drwxr-xr-x    7 root     root          4096 Mar 23 20:43 src
drwxr-xr-x    3 root     root          4096 Mar 23 20:43 static
```

only files related to the git are not ignored (which also have a dot).

I use this action in open source, so if it somehow helps, here's the repository where it is used: https://github.com/Chronoblog/gatsby-theme-chronoblog

**Comment by cpina** [comment]
Oops, I need to look at this and some other issues soon (main/master thing). Hopefully next weekend I should have some time to address these things and do a new release (if not before). Thanks for the issue!

**Comment by cpina** [comment]
Hi @Ganevru

I think that I see what's happening (actually two things are happening! sorry about that!). The files that you see listed by the GitHub Action are the files from the destination directory (and this is confusing, I'll make some changes to clarify this in the next version).

The files that should be copied are the files (in your examples) in `examples/chronoblog-netlify-cms/`.

I've done in my computer:
```
git clone https://github.com/Chronoblog/gatsby-theme-chronoblog
cd gatsby-theme-chronoblog/
npm i
npm run generate-starters
ls -la examples/chronoblog-netlify-cms/
```
I don't see .eslintrc.json neither .prettierrc.json so they are not copied (I do see .prettierignore and it should be copied... keep reading)

About the .prettierignore and maybe the .eslintrc.json and .prettierrc.json (if they existed for you but not for me):
This was fixed in `master` branch but not released in `v1.2` which is the one that you are using.

Could you try with `master` instead of `v1.2`? It should copy the files starting with "." (if they are there, and they might be there for you!).

Thanks!

**Comment by Ganevru** [comment]
Yes, using `master` works! Thank you.

About `.eslintrc.json `and `.prettierrc.json` - I deleted them and transferred their functionality to files without dots. I left only `.prettierignore` as prettier has no way to use a different ignore file format. This was done in order not to confuse users.
